### ISSUE TITLE
jQuery problems in exception render

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#6172]
Under certain circum stances, exceptions can be thrown **before** jQuery is sent, leaving this in the console:
```
Uncaught ReferenceError: $ is not defined
    at HTMLAnchorElement.onclick (my-page:49)
```

…and the exception trace unusable.

Is it possible to not use a library for this?

**Comment by rarila** [comment]
Sure, question is where is ``my-page:49`` in code?

And is it possible to enforce this error by adding some errornous php code?

**Comment by GawainLynch** [comment]
That's the thing… the page never gets rendered, so you can't check the HTML… but it is just a `entry.twig` from `base-2016` … the onClick is listening on the attributes for the exception.

It's a  hard one to reproduce at the moment, but completely kills the use of the exception page hence the :fire: jQuery :fire: … it has to go.

**Comment by bobdenotter** [comment]
This is not a problem with Jquery per se. I'll look into it.

**Comment by bobdenotter** [comment]
Closed by #6184
